lib/bolt/transport/winrm.rb

```diff
@@ -100,8 +100,17 @@ def run_script(target, script, arguments, _options = {})
       end
 
       def run_task(target, task, arguments, _options = {})
-        executable = target.select_impl(task, PROVIDED_FEATURES)
-        raise "No suitable implementation of #{task.name} for #{target.name}" unless executable
+        if from_api?(task)
+          # TODO: Remove as part of BOLT-664
+          executable = File.join(__dir__, task.file['filename'])
```

Comment:
That's definitely not a great place to put it. I'd rather use mktmpdir to ensure we don't accidentally overwrite part of Bolt.